charts/headlamp/values.yaml

```diff
@@ -96,6 +96,7 @@ config:
       name: ""
   # -- directory to look for plugins
   pluginsDir: "/headlamp/plugins"
+  enableHelm: false
```

Comment:
We’ll need to add an entry to the documentation for this variable in charts/headlamp/README.md